backend/Makefile

```diff
@@ -12,24 +12,39 @@
 OUT_DIR = dist/
```

Comment:
This comment is to hold merging until dependent pr is merged. That PR will also have a merge conflict once dependent PR is merged. Once OS changes are in feel free to close this comment